app/services/analytics_events.rb

```diff
@@ -664,6 +664,17 @@ def idv_in_person_prepare_submitted(flow_path:, in_person_cta_variant:, **extra)
     )
   end
 
+  # @param [String] nontransliterable_characters
+  # Nontransliterable characters submitted by user
+  def idv_in_person_proofing_nontransliterable_characters_submitted(nontransliterable_characters:,
+                                                                    **extra)
```

Comment:
I think splitting params onto the next lines would be easier to follow
```suggestion
  def idv_in_person_proofing_nontransliterable_characters_submitted(
    nontransliterable_characters:,
    **extra
  )
```